modules/jlink/artifacts/opt/jboss/container/java/jlink/preflight.sh

```diff
@@ -0,0 +1,12 @@
+jlink_preflight_check()
+{
+    # preflight check: do we have what we need?
+    if [ "$JAVA_VERSION" -lt 11 ]; then
+      echo "Jlink integration not available for JDK${JAVA_VERSION}!"
+      echo "Jlink integration is only supported for JDK versions 11 and newer."
```

Comment:
I feel if the JAVA_VERSION is less than 11 then we can stop doing the further steps and exit here itself with the above message. WDYT?

Reply:
Yes; there should be an `exit 1` here, and also in the fail branch below. Thanks!